lib/salestation/web/statsd_middleware.rb

```diff
@@ -27,9 +27,10 @@ def call(env)
           end
 
         tags = [
-          "path:#{ path }",
-          "method:#{ method }",
-          "status:#{ status }"
+          "path:#{path}",
+          "method:#{method}",
+          "status:#{status}",
+          "status_class:#{status / 100}xx"
```

Comment:
is the decimal automatically floored?

Reply:
seems so from tests :+1:

Reply:
Isn't it floored automatically because status is an integer and not a float?

Reply:
it is